### ISSUE TITLE
Django settings import

### ISSUE THREAD

**Issue by k0t3n** [rocioar/flake8-django#91]
Detect wrong django settings import:
```python
from .settings import DEBUG  # should detect
from django.conf import settings  # ok
```

**Comment by CalumY** [comment]
This can be fixed using [flake8-tidy-imports](https://github.com/adamchainz/flake8-tidy-imports) using the `banned-modules` configuration.
